src/wire/ip.rs

```diff
@@ -1,5 +1,7 @@
 use core::fmt;
 use core::convert::From;
+#[cfg(all(feature = "std", feature = "proto-ipv4", feature = "proto-ipv6"))]
```

Comment:
I'd prefer if we had no conditional `use` statements.

Reply:
I've changed it to use global paths instead. eg. `::std::net::Ipv4Addr`.